pkg/operator/nfs/controller_test.go

```diff
@@ -1,182 +0,0 @@
-/*
-Copyright 2018 The Rook Authors. All rights reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-package nfs
-
-import (
-	"fmt"
-	"strings"
-	"testing"
-
-	nfsv1alpha1 "github.com/rook/rook/pkg/apis/nfs.rook.io/v1alpha1"
-	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/operator/k8sutil"
-	testop "github.com/rook/rook/pkg/operator/test"
-	"github.com/stretchr/testify/assert"
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes/fake"
-)
-
-const appName = "nfs-server-X"
-
-func TestValidateNFSServerSpec(t *testing.T) {
```

Comment:
Unit tests needs to be added.

Reply:
initially i planned to add unit test as follow up later, but if needed as part of this PR i'll add the unit test soon

Reply:
@rohan47 @jbw976 
i've just added new commit for unit test